go/libkb/rpc_exim.go

```diff
@@ -915,7 +915,7 @@ func publicKeyV2BaseFromComputedKeyInfo(info ComputedKeyInfo) (base keybase1.Pub
 	}
 	if info.DelegatedAt != nil {
 		base.Provisioning = keybase1.SignatureMetadata{
-			Time: keybase1.Time(info.DelegatedAt.Unix),
+			Time: keybase1.TimeFromSeconds(info.DelegatedAt.Unix),
```

Comment:
Converting to UpakV2 revealed a couple bugs in my converter function. The first is that I wasn't converting these two times properly. The second was that I was assembling an unsorted tracks list, which broke a `sort.Search` that happens later (that took forever to figure out :p).